raiden/ui/cli.py

```diff
@@ -682,7 +682,7 @@ def run(ctx: Context, **kwargs: Any) -> None:
         name_or_id = ID_TO_NETWORKNAME.get(kwargs["network_id"], kwargs["network_id"])
         click.secho(
             f"FATAL: Another Raiden instance already running for account "
-            f"{to_checksum_address(address)} on network id {name_or_id}",
+            f"address on network id {name_or_id}",
```

Comment:
Is this intentional? We won't display the account afterwards.

Reply:
thanks for the catch, should have been
```suggestion
            f"{address} on network id {name_or_id}",
```